commit_screenshots.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
Can you please add a copyright header for this file? The copyright owner would be yourself or your employer, depending on your specific situation. The license should be the same as in the repo (Apache 2.0)

Reply:
Added